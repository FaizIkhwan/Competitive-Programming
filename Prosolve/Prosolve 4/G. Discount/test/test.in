5
0
100
1000
1337
5000
