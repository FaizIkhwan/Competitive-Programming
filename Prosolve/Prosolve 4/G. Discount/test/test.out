RM 0.00
RM 100.00
RM 950.00
RM 1230.04
RM 4250.00
